src/js/components/Table.js

```diff
@@ -50,6 +52,9 @@ var Table = React.createClass({
     if (element && element.parentNode.nodeName === 'TBODY') {
       this._clearSelection();
       element.classList.add(CLASS_ROOT + "__row--selected");
+      if (this.props.onSelect) {
+        this.props.onSelect(element);
```

Comment:
Thanks for the pull request.

Given that the selection property is a row index number, it would be more symmetric if the onSelect function was passed the row index number that was selected.